src/models/record.ts

```diff
@@ -103,6 +118,9 @@ export default class Record extends AbstractDomainEntity {
 
     static nextStage(record:Record):string {
         var stages = ["upload","validation","data-service"];
+        if(record.type == 'osw'){
```

Comment:
make common method to get stages passing Record param, as this is used two places